src/Utils/cite.worker.ts

```diff
@@ -0,0 +1,126 @@
+import axios from "axios";
+import CSL from "citeproc";
+var parseString = require("xml2js").parseString;
+
+const ctx: Worker = self as any;
+var itemsObj = {};
+var style = null;
+var localesObj = null;
+var preferredLocale = null;
+var citeproc = null;
+var citationByIndex = null;
+var citationData = [];
+
+const sys = {
+  retrieveItem: function (itemID: string | number) {
+    return itemsObj[itemID];
+  },
+  retrieveLocale: function (locale: string | number) {
+    return localesObj[locale];
+  },
+};
+
+async function getFileContent(type: string, filename: string) {
+  if (type === "styles") {
+    filename = filename + ".csl";
+  } else if (type === "locales") {
+    filename = "locales-" + filename + ".xml";
+  }
+  const url = "../data/" + type + "/" + filename;
+  console.log("axios call", url);
+  const resp = await fetch(url);
+  console.log(resp.text());
+  return resp.text();
```

Comment:
Don't you need an `await` for `resp.text` as well? Other than that, what is it that isn't working?